src/components/pages/TeachersPage/Conversation.tsx

```diff
@@ -6,13 +6,13 @@ import { Box, Typography } from '@mui/material';
 import conversationCSS from './Conversation.css';
 import { filterWords, scrollDown } from '@utils/classrooms';
 
-export default function Conversation({ chat }) {
+export default function Conversation({ chat, parentRef }) {
   const [student1, student2] = chat.studentPair;
   const lastMessage = useRef(null);
 
   useEffect(() => {
-    scrollDown(lastMessage);
-  }, [chat.conversation]);
+    scrollDown(lastMessage, parentRef);
```

Comment:
I don't think we need the `parentRef`.  I think we could just use an `if` statement to make sure it's not the homepage chat. For example:
```suggestion
   if (chat.chatId !== 'homepage sample chat") scrollDown(lastMessage);
```
And we'd change the hardcoded chatId for the homepage's chat to the above.

Reply:
Fair enough, will do.